README.md

```diff
@@ -91,6 +90,8 @@ end-patch
 remove frame_1/DoAction.as 789-1111
 ```
 
+**Note:** A patch file may have a .assets file extension, but this is deprecated and should only be used for backwards compatibility.
```

Comment:
Probably not necessary. Not enough people have used the patcher yet to warrant issuing a correction, in my eyes.

Reply:
We should keep this for backward compatibility, for people (like SMF) who use the older patcher format.